tests/Backend/CommonPart1/AlterTableTest.php

```diff
@@ -211,11 +211,15 @@ public function testTooManyColumns(): void
         $importFile = __DIR__ . '/../../_data/many-more-columns.csv';
 
         $tokenData = $this->_client->verifyToken();
-        // TODO should be also checked for Snowflake, but it has it manually in Connection
+
         if ($tokenData['owner']['defaultBackend'] == self::BACKEND_EXASOL) {
             $this->markTestSkipped('Exasol backend does not have any limit');
         }
 
+        if ($tokenData['owner']['defaultBackend'] == self::BACKEND_SNOWFLAKE) {
+            $this->markTestSkipped('Snowflake backend does not have any limit');
+        }
```

Comment:
Pokud SNFLK ten limit nemá, tak by bylo fajn se k tomu postavit tak, že otestuješ, že teda těch 5000 column bude fungovat. A ne ten test skipnout. Doteď to házelo exception, odteď to má nedefinované chování ;)

Reply:
rusim, nakonec tam ten limit nechavam, jen jsem ho v connection presunul jinam.